frontend/templates/posts/list.html

```diff
@@ -0,0 +1,78 @@
+{% extends "layout.html" %}
+{% block title %}light and sound{% endblock %}
```

Comment:
Title field is not consistent between files

Neither is use of header or styles blocks